tests/eosio.bios_inst_fin_tests.cpp

```diff
@@ -55,12 +56,11 @@ BOOST_FIXTURE_TEST_CASE( set_1_finalizer, eosio_bios_if_tester ) try {
     abi_serializer::to_variant( *cur_block, pretty_output, get_resolver(), fc::microseconds::maximum() );
 
     BOOST_REQUIRE(pretty_output.get_object().contains("instant_finality_extension"));
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["generation"], 1);
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["threshold"], 2);
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["finalizers"].size(), 1u);
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["finalizers"][size_t(0)]["description"], "set_1_finalizer");
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["finalizers"][size_t(0)]["weight"], 2);
-    BOOST_REQUIRE_EQUAL(pretty_output["instant_finality_extension"]["new_finalizer_policy"]["finalizers"][size_t(0)]["public_key"], "PUB_BLS_6j4Y3LfsRiBxY-DgvqrZNMCttHftBQPIWwDiN2CMhHWULjN1nGwM1O_nEEJefqwAG4X09n4Kdt4a1mfZ1ES1cLGjQo6uLLSloiVW4i9BUhMHU2nVujP1_U_9ihdI3egZ17N-iA");
+    std::string output_json = fc::json::to_pretty_string(pretty_output);
+    BOOST_TEST(output_json.find("\"generation\": 1") != std::string::npos);
```

Comment:
Why not use `BOOST_REQUIRE_NE`?

Reply:
My understanding is `BOOST_TEST`, `BOOST_TEST_<level>` are encouraged over the old `BOOST_<level>` macros. These do not require a REQUIRE.
https://live.boost.org/doc/libs/1_85_0/libs/test/doc/html/boost_test/testing_tools/boost_test_universal_macro.html